Rakefile

```diff
@@ -57,24 +57,17 @@ end
 task :build, [:build_fat] do |t, args|
   args.with_defaults(:build_fat => false)
   print_info "Building project (fat: #{args[:build_fat]})"
-  sh %Q(swift build -c release --arch x86_64 --arch arm64 --disable-sandbox --build-path #{BUILD_DIR})
+
+  # Use xcodebuild due to https://bugs.swift.org/browse/SR-15802
+  xcpretty %Q(xcodebuild -scheme sourcery -destination generic/platform=macOS -archivePath #{BUILD_DIR}sourcery.xcarchive archive)
```

Comment:
So the one downside to using a static library distributed via XCFramework's is [SR-15802](https://bugs.swift.org/browse/SR-15802). `swift build`/`swift run` will work fine, but specifying `--arch x86_64 --arch arm64` will result in duplicate symbols. 

To work around this, I updated the `build` task to use `xcodebuild` instead but are you happy with this? I don't know if we need to be able to build a fat binary anywhere else? Will it cause problems elsewhere by not being able to use `swift build -c release --arch x86_64 --arch arm64`? Is it worth it? 

We could _maybe_ do something hacky in **Package.swift** to work around this, but I'm not sure how that would look so maybe its better to avoid it if possible for now.

Reply:
is this still building an universal binary? if so then I guess it's ok